### PR TITLE
Fixed some typo's/grammer/style issues

### PR DESCRIPTION
**Pull request by Kriskras99**

### CODE REVIEW

File: book_src/Operating System From 0 to 1.lyx [hunk 1]
```diff
@@ -5074,7 +5074,7 @@ execute
  More instructions can be implemented by adding more devices and allocating
  more opcodes for the instructions, then update the decoder accordingly.
  The Apollo Guidance Computer, a digital computer produced for the Apollo
- space program from 1961 -- 1972, was built entirely with a NOR gate - the
+ space program from 1961 -- 1972, was built entirely with NOR gates - the
```
Comment (tuhdo) [inline book_src/Operating System From 0 to 1.lyx]: Wow thanks for spotting this.

**Comment by tuhdo** [comment]
Thanks for improving my ENnglish!

File: book_src/Operating System From 0 to 1.lyx (hunk 1 above)
Comment (Kriskras99) [inline book_src/Operating System From 0 to 1.lyx]: No problem. It's a great book.

**Comment by Kriskras99** [comment]
And you for writing this book!

**Comment by tuhdo** [comment]
😄

**Pull request merged**